backend/dataline/services/llm_flow/toolkit.py

```diff
@@ -64,14 +59,13 @@ def execute_sql_query(
     db: SQLDatabase, query: str, for_chart: bool = False, chart_type: Optional[ChartType] = None
 ) -> QueryRunData:
     """Execute the SQL query and return the results or an error message."""
-    result = cast(Result[Any], db.run(query, fetch="cursor", include_columns=True))  # type: ignore[misc]
+    columns, rows = db.custom_run_sql(query)
```

Comment:
Why use a custom method here? Why not just override the original and keep the same API for consistency?

Reply:
We can't override the `run` method in a clean way because if the dialect is mssql, we can't return a cursor and are forced to return the rows and columns. So if we override the method, it'll have different outputs than its parent implementation and the method args lose their effect (eg. `fetch`). If we're fine with that, we can override the parent method, but I thought this was a bit cleaner and more specific